sql-db/narayana-transactions/src/test/java/io/quarkus/ts/transactions/DevModeOpenTelemetryIT.java

```diff
@@ -35,7 +35,7 @@ public class DevModeOpenTelemetryIT {
 
     @DevModeQuarkusApplication
     static DevModeQuarkusService app = (DevModeQuarkusService) new DevModeQuarkusService()
-            .withProperty("quarkus.otel.exporter.otlp.traces.endpoint", jaeger::getCollectorUrl)
+            .withProperty("quarkus.otel.exporter.otlp.traces.legacy-endpoint", jaeger::getCollectorUrl)
```

Comment:
I can't make sense of this change as on other places, you make changes to use `quarkus.otel.exporter.otlp.traces.endpoint` that now are removing here..